python/tank/util/shotgun.py

```diff
@@ -264,9 +264,7 @@ def get_associated_sg_base_url():
     
     :returns: The base url for the associated Shotgun site
     """
-    cfg = __get_sg_config()
-    config_data = __get_sg_config_data(cfg)
-    return config_data["host"]
+    return get_associated_sg_config_data().get("host", "")
```

Comment:
This is necessary because at the moment the authentication code is dependant on a configured core. When we switch the code to an independent authentication module I believe this can go. I will add a FIXME indicating this needs to be put back as it is. Good point.